src/ptvsd/_vendored/pydevd/build_tools/build.py

```diff
@@ -95,22 +96,21 @@ def build():
 
     os.chdir(root_dir)
 
-    env=None
+    env = None
     if sys.platform == 'win32':
         # "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\vcvars64.bat"
         # set MSSdk=1
         # set DISTUTILS_USE_SDK=1
         # set VS100COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 9.0\Common7\Tools
 
-
         env = os.environ.copy()
-        if sys.version_info[:2] in ((2, 6), (2, 7), (3, 5), (3, 6), (3, 7)):
-            import setuptools # We have to import it first for the compiler to be found
+        if sys.version_info[:2] in ((2, 6), (2, 7), (3, 5), (3, 6), (3, 7), (3, 8)):
```

Comment:
Note: added the changes related to building in Python 3.8 in this pull request (they're needed for updating the binaries on Python 3.8).